Emby.Server.Implementations/Playlists/PlaylistManager.cs

```diff
@@ -135,16 +135,8 @@ public async Task<PlaylistCreationResult> CreatePlaylist(PlaylistCreationRequest
                 {
                     Name = name,
                     Path = path,
-                    Shares = new[]
-                    {
-                        new Share
-                        {
-                            UserId = options.UserId.Equals(default)
-                                ? null
-                                : options.UserId.ToString("N", CultureInfo.InvariantCulture),
-                            CanEdit = true
-                        }
-                    }
+                    OwnerUserId = options.UserId,
+                    Shares = options.Shares
```

Comment:
```suggestion
                    Shares = options.Shares ?? Array.Empty<Share>();
```
`PlaylistCreationRequest.Shares` is nullable, `IHasShares.Shares` is not. When creating a new playlist on your branch I was getting null exceptions iterating `item.Shares` in `BaseXmlSaver.AddSharesAsync` without this line.